src/pages/wallets/find-wallet.tsx

```diff
@@ -216,28 +232,33 @@ const FindWalletPage = ({
         <Box px={{ md: 4, "2xl": 0 }}>
           <Flex pt={4} pb={6} gap={6}>
             {/* Filters sidebar */}
-            <WalletFilterSidebar
-              hideBelow="lg"
-              top={calc(NAV_BAR_PX_HEIGHT).subtract("2px").toString()}
-              {...{
-                filters,
-                resetWalletFilter,
-                updateFilterOption,
-                updateFilterOptions,
-                resetFilters,
-                selectedPersona,
-                setFilters,
-                setSelectedPersona,
-              }}
-            />
+            {/* Use `Show` instead of `hideBelow` prop to avoid rendering the sidebar on mobile */}
+            <Show above="lg">
```

Comment:
This will add a bit of CLS on the table section on desktop versions. I think it is worth it considering that we are removing this from the render of slower mobile devices.
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/fb0b0909-1060-4c8e-b147-7623dee8257b)

Reply:
Okay.. yeah, if we're preventing a large unnecessary chunk of data from loading then I think it's worth it.